alacritty/src/config/ui_config.rs

```diff
@@ -169,3 +195,141 @@ pub struct Delta<T: Default> {
     /// Vertical change.
     pub y: T,
 }
+
+/// Regex terminal hints.
+#[derive(ConfigDeserialize, Default, Debug, PartialEq, Eq)]
+pub struct Hints {
```

Comment:
Might move this to some other file as well.

Reply:
Not sure why this marked as resolved, since the hints code is still in ui_config, and not in config/hints.rs(e.g.).